### PR TITLE
Added Win UI 3 support

### PR DESCRIPTION
**Pull request by dahovey**
_**Edited this original comment, since Windows App SDK 1.0 has been released. At the time I created the original PR it hadn't been released yet, and `Project Reunion 0.8.5` was used.**_

### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Feature. Adding Win UI 3 support using Windows App SDK 1.0.

### :boom: Does this PR introduce a breaking change?

No. Just adding new projects. This will involve adding a new framework moniker/dlls to the MvvmCross cross NuGet package for targeting .NET 5 and WinUI 3. 

### :bug: Recommendations for testing

Need to install "Windows App SDK" Extension in Visual Studio 2019/2022. I added `Playground.WinUi` for testing.

**Some notes about the changes:**

- To start with I copied the `Uap` platform folder and renamed it to `WinUi` as a starting point since Win UI 3 is based upon Win UI 2 which UWP uses. I renamed namespaces within and changed references to `Windows.UI.Xaml` to `Microsoft.UI.Xaml` namespace. 
- The `Dispatcher` property of `Window` is null in WinUI 3 so updated `MvxWindowsMainThreadDispatcher` to use `DispatcherQueue` instead per [here](https://github.com/microsoft/microsoft-ui-xaml/issues/4219).
- Removed 'entering background', 'suspension', 'resuming' event handling functions in `MvxApplication` class. My understanding is that Win UI 3 is moving away from sandbox environment of UWP to desktop app lifecycle so those events aren't available.

Very likely there is more clean-up to do, since it appears Win UI 3 includes properties that are only applicable within the context of a UWP application. My thought is that any code that would only apply in UWP could be removed, since Microsoft have deprecated UWP. So if a developer would like to use MvvmCross with UWP they can do so, but using Win UI 2, as MvvmCross currently supports.

I did not tackle updating any documentation since this is quite a large PR anyway. Please advise how best to proceed. I can add documentation under this PR, but I wanted to get feedback on what I have so far.

### :memo: Links to relevant issues/docs

Issue #3816. 

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contributing/mvvmcross-docs-style-guide))
- [x] Rebased onto current develop

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross?pullRequest=4316) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross?pullRequest=4316) before we can accept your contribution.<br/><br/>:x: dahovey [sign now](https://cla.dotnetfoundation.org/MvvmCross/MvvmCross?pullRequest=4316)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/MvvmCross/MvvmCross?pullRequest=4316) it.</sub>

**Comment by dahovey** [comment]
@Cheesebaron I added .NET 5 support in the pipeline, so the build gets farther than it did, but the build is reporting weird error that appears it can't find the Playground.Core assembly. I am not familiar enough with the build pipeline to understand why that could be.

**Comment by Cheesebaron** [comment]
Sorry, been down with a nasty cold recently. Hence, the late reply.

Thank you for your contribution. It is a pretty big one, so will need to spend a bit of time testing it out.

I have no experience with WinUI, so I need to learn this a bit to understand what is what. However, at a glance it looks OK to me.

**Comment by dahovey** [comment]
No worries. Glad to have you back.

**Comment by afernandes** [comment]
Hey guys! Any news?

**Comment by afernandes** [comment]
image
image

**Comment by dahovey** [comment]
@Cheesebaron Is there anything we can do to move this along?

@afernandes Can you clarify if you found an issue? To me your screenshots aren't clear.

**Comment by afernandes** [comment]
> @Cheesebaron Is there anything we can do to move this along?
> 
> @afernandes Can you clarify if you found an issue? To me your screenshots aren't clear.

my print is just a sample of how to apply the latest WinUI 3 theme.
Change the file: App.xaml

**Comment by afernandes** [comment]
@Cheesebaron
Changing the csproj this way worked for me.
image

It also includes these files, but I don't remember now if they are really needed
image

File: Projects/Playground/Playground.WinUi3/Playground.WinUi3.csproj [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <RootNamespace>Playground.WinUi3</RootNamespace>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+    <UseWinUI>true</UseWinUI>
+    <EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
+    <EnableDefaultPageItems>false</EnableDefaultPageItems>
```
Comment (afernandes) [inline Projects/Playground/Playground.WinUi3/Playground.WinUi3.csproj:12]:
```
<!-- This bundles the .NET Core libraries -->
<SelfContained>true</SelfContained>
<!-- This bundles the WinUI3 components -->
<WindowsAppSDKSelfContained>true</WindowsAppSDKSelfContained>
```

**Comment by Cheesebaron** [comment]
> @Cheesebaron Changing the csproj this way worked for me. image
> 
> It also includes these files, but I don't remember now if they are really needed image

That doesn't seem to help at all. After also adding RuntimeIdentifier, I get:

 ```
8>Build started 02/04/2022 18.29.56.
8>Target _GetProjectReferenceTargetFrameworkProperties:
8>Target ResolveProjectReferences:
8>Target GetReferenceTargetPlatformMonikers:
8>Target MarkupCompilePass1:
8>  (Out) GeneratedCodeFiles: 'C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\XamlTypeInfo.g.cs'
8>  (Out) GeneratedXamlFiles: 'C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.xaml'
8>  (Out) GeneratedXbfFiles: 'C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.xbf'
8>  (Out) Compile: 'App.xaml.cs;Views\ChildView.xaml.cs;Views\DialogView.xaml.cs;Views\RootView.xaml.cs;Views\SecondChildView.xaml.cs;Views\SplitDetailView.xaml.cs;Views\SplitMasterView.xaml.cs;Views\SplitRootView.xaml.cs;WinUiPlaygroundSetup.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\XamlTypeInfo.g.cs'
8>Target XamlPreCompile:
8>  CompilerServer: server - server processed compilation - 4ad2d0cf-f19f-4b81-8439-c59cdc40f41c
8>Target MarkupCompilePass2:
8>  (Out) GeneratedCodeFiles: 'C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\XamlTypeInfo.g.cs'
8>  (Out) GeneratedXamlFiles: 'C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.xaml;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.xaml'
8>  (Out) GeneratedXbfFiles: 'C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.xbf;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.xbf'
8>  (Out) GeneratedXamlPagesFiles: ''
8>  (Out) Compile: 'App.xaml.cs;Views\ChildView.xaml.cs;Views\DialogView.xaml.cs;Views\RootView.xaml.cs;Views\SecondChildView.xaml.cs;Views\SplitDetailView.xaml.cs;Views\SplitMasterView.xaml.cs;Views\SplitRootView.xaml.cs;WinUiPlaygroundSetup.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\App.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\ChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\DialogView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\RootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SecondChildView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitDetailView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitMasterView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.i.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\Views\SplitRootView.g.cs;C:\git\MvvmCross\Projects\Playground\Playground.WinUi3\obj\x64\Debug\net5.0-windows10.0.19041.0\win10-x64\XamlTypeInfo.g.cs'
8>Target MergeWindowsAppSDKManifests:
8>  The system cannot find the path specified.
8>  C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\build\Microsoft.WindowsAppSDK.SelfContained.targets(19,5): error MSB3073: The command ""C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\mt.exe" -nologo -manifest app.manifest app.manifest C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\manifests\Microsoft.InteractiveExperiences.manifest C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\manifests\Microsoft.WindowsAppSdk.Foundation.manifest C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\manifests\Microsoft.WindowsAppSdk.WinUI.manifest -out:Microsoft.WindowsAppSdk.manifest" exited with code 3.
8>Done building target "MergeWindowsAppSDKManifests" in project "Playground.WinUi3.csproj" -- FAILED.
8>
8>Done building project "Playground.WinUi3.csproj" -- FAILED.
8>
8>Build FAILED.
8>
8>C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\build\Microsoft.WindowsAppSDK.SelfContained.targets(19,5): error MSB3073: The command ""C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\mt.exe" -nologo -manifest app.manifest app.manifest C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\manifests\Microsoft.InteractiveExperiences.manifest C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\manifests\Microsoft.WindowsAppSdk.Foundation.manifest C:\Users\Cheesebaron\.nuget\packages\microsoft.windowsappsdk\1.0.0\manifests\Microsoft.WindowsAppSdk.WinUI.manifest -out:Microsoft.WindowsAppSdk.manifest" exited with code 3.
8>    0 Warning(s)
8>    1 Error(s)
8>
8>Time Elapsed 00:00:03.43
========== Build: 7 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

**Comment by dahovey** [comment]
@Cheesebaron, @afernandes  For the build / packaging errors you received, you will also need VS2022 extension:
[Single-project MSIX Packaging Tools for VS 2022](https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftSingleProjectMSIXPackagingToolsDev17)

Documentation [here](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/single-project-msix?tabs=csharp)

**Comment by dahovey** [comment]
@Cheesebaron This is again, ready for your review. Thanks for your patience.

**Comment by Cheesebaron** [comment]
@dahovey thanks, will have a look. I guess we would like this even more now that I removed some of the old TFMs :)

**Pull request merged**